zip-proportional-output-fee-mechanism-pofm.rst

```diff
@@ -0,0 +1,195 @@
+::
+
+  ZIP: XXX
+  Title: Proportional Output Fee Mechanism (POFM)
```

Comment:
```suggestion
  Title: Proportional Output Fee Mechanism
```
ZIP titles typically only have the spelled-out name (compare ZIP 222 which is just "Transparent Zcash Extensions").

Reply:
Also, why is it called "Proportional *Output* Fee Mechanism", when the fee is (roughly) proportional to the total number of inputs and outputs?

Reply:
Okay, I noticed the acronym in the title set as per this ZIP https://github.com/zcash/zips/blob/main/zip-0309.rst

And `Output` keyword is nuanced to convey the result of the fee calculation. Please feel free to suggest a new title.

Reply:
Here's one possible alternative name:

```suggestion
  Title: Proportional Transfer Fee Mechanism
```